components/script/dom/mutationrecord.rs

```diff
@@ -43,6 +43,21 @@ impl MutationRecord {
         reflect_dom_object(record, &*window_from_node(target), MutationRecordBinding::Wrap)
     }
 
+    pub fn character_data_mutated(target: &Node,
```

Comment:
nit: Please indent it with block indentation:

```rust
pub fn character_data_mutated(
    target: &Node,
    old_value: Option<DOMString>,
) -> DomRoot<MutationRecord> {
```

Reply:
I followed the local style in this file.

Reply:
Some of our files are still using non-standard Rust styling, since they were merged before we had proper styling through rustfmt, and we're refactoring them as we spot them.

Reply:
And in these cases you refactor the whole file I guess?

Reply:
not necessarily... Feel free to fix it or not though.